client/src/App.js

```diff
@@ -25,6 +26,7 @@ import './styles/styles.css';
     <Route path="/calculator" element={<Calculator />} />
     <Route path="/create" element={<Create />} />
     <Route path="/edit/:id" element={<Edit />} />
+    <Route path="/view/:id" element={<View />} />
```

Comment:
Look at main branch for the PageFramework implementation I added. I think you can just insert <View/> as the component parameter into PageFramework.